src/util/Permissions.js

```diff
@@ -7,7 +7,7 @@ const { RangeError } = require('../errors');
  */
 class Permissions {
   /**
-   * @param {number|PermissionResolvable[]} permissions Permissions or bitfield to read from
+   * @param {number|PermissionResolvable|PermissionResolvable[]} permissions Permission(s) or bitfield to read from
```

Comment:
number is included in PermissionResolvable...

Reply:
`number` was in the JSDoc before I started working on this. I'll remove it now though.